Hardware/Hardware/Printer/ReceiptContent.swift

```diff
@@ -57,3 +58,15 @@ public extension ReceiptContent {
         )
     }
 }
+
+public extension ReceiptContent {
+    static let pointsPerInch: Int = 72
+
+    /// Returns the preferred page size for a receipt in points. There are 72 points per inch.
+    /// In the future, we could calculate this based on the receipt content. For now, let's
+    /// just return a size that should accomodate the vast majority of receipts.
+    ///
+    var preferredPageSizeForPrinting: CGSize {
+        return CGSize(width: 4 * ReceiptContent.pointsPerInch, height: 10 * ReceiptContent.pointsPerInch)
```

Comment:
I understand this leaves enough vertical space for receipts to grow, but can we at least log a ticket to estimate a height, please?